docs/install.md

```diff
@@ -79,6 +79,57 @@ Output:
 +---------+---------------+--------------------+-----------------+
 ```
 
+## Updating OpenCost
+To update your OpenCost to the most recent version, using a previously unmodified opencost.yaml manifest, enter the following command. This will update OpenCost to the latest version.
+
+```sh
+$ kubectl apply -f opencost.yaml -n opencost
+namespace/opencost created
+serviceaccount/opencost created
+clusterrole.rbac.authorization.k8s.io/opencost created
+clusterrolebinding.rbac.authorization.k8s.io/opencost created
+deployment.apps/opencost created
+service/opencost created
+```
+
+To check logs to verify the version of your OpenCost:
+
+```sh
+$  kubectl logs -n opencost deployment/opencost | head
+2022-09-02T18:20:34.327989163Z ??? Log level set to info
+2022-09-02T18:20:34.328206357Z INF Starting cost-model (git commit "x.xx.x")
+```
+
+### Sidegrading OpenCost
+If you wish to modify OpenCost to a previous version, start with the following command in order to modify the opencost.yaml manifest:
+
+`$ vim opencost.yaml`
+
+In the line `quay.io/kubecost1/kubecost-cost-model:latest`, change `latest` to the desired version number in the format `prod-x.xx.x`. Then enter the following command to apply the updated opencost.yaml manifest:
+
+```sh
+$ kubectl apply -f opencost.yaml -n opencost         
+namespace/opencost unchanged
+serviceaccount/opencost unchanged
+clusterrole.rbac.authorization.k8s.io/opencost unchanged
+clusterrolebinding.rbac.authorization.k8s.io/opencost unchanged
+deployment.apps/opencost configured
+service/opencost unchanged
+```
+
+Finally, check the logs to verify the version of your OpenCost has successfully been changed:
+
+```sh
+$  kubectl logs -n opencost deployment/opencost | head
+2022-09-02T18:20:34.327989163Z ??? Log level set to info
+2022-09-02T18:20:34.328206357Z INF Starting cost-model (git commit "x.xx.x")
+```
+
+## Deleting OpenCost
+To delete OpenCost, enter the following command:
+
+`kubectl delete pod opencost -n opencost`
```

Comment:
This will only kill the pod, and it will be recreated because it is a dpeloyment.
`kubectl delete -f https://raw.githubusercontent.com/opencost/opencost/develop/kubernetes/opencost.yaml`
will clean up everything.